docs/conceptual-framework2.md

```diff
@@ -182,6 +182,20 @@ This sorting operation corresponds to top-_k_ retrieval.
 We can see that the output is the same as search with `FaissSearcher` above.
 This is exactly as expected.
 
+<details>
+<summary>Trying it with Contriever:</summary>
+<br/>
+
+To repeat with Contriever, replace:
```

Comment:
Can we just have the entire snippet of code instead of replacing _X_ with _Y_?

Also, report expected response?

Reply:
done